src/types/shared.ts

```diff
@@ -141,6 +141,22 @@ export interface KlinesParams {
   endTime?: number;
   limit?: number;
 }
+
+export type Kline = [
+  number, // open time
+  numberInString, // open
+  numberInString, // high
+  numberInString, // low
+  numberInString, // close
+  numberInString, // volume
+  number, // close time
+  numberInString, // quote asset volume
+  number, // number of trades
+  numberInString, // taker buy base asset vol
+  numberInString, // taker buy quote asset vol
+  numberInString // ignore?
+];
+
```

Comment:
```suggestion
export type Kline = [
  number, // open time
  numberInString, // open
  numberInString, // high
  numberInString, // low
  numberInString, // close
  numberInString, // volume
  number, // close time
  numberInString, // quote asset volume
  number, // number of trades
  numberInString, // taker buy base asset vol
  numberInString, // taker buy quote asset vol
  numberInString // ignore?
];

/** @deprecated `FuturesKline` will be removed soon. Use `Kline` instead. **/
export type FuturesKline = Kline;
```

This is great, thank you! Tiny nitpick, could you add a backwards compatible type like this preserving the old name?

By removing the old one you may be causing a breaking change, and I'd rather avoid those for minor patch releases.